### PR TITLE
fix location of helm-release-crd

### PR DESCRIPTION
**Pull request by dholbach**
Thanks @himpich for spotting this!

Does `chart/flux/README.md` need updating too?

### CODE REVIEW

**Comment by dholbach** [comment]
https://github.com/fluxcd/helm-operator-get-started too

File: docs/tutorials/driving-flux.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ helm repo add fluxcd https://charts.fluxcd.io
 Apply the Helm Release CRD:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/fluxcd/flux/master/deploy-helm/flux-helm-release-crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/fluxcd/helm-operator/master/deploy/flux-helm-release-crd.yaml
```
Comment (stefanprodan) [inline docs/tutorials/driving-flux.md]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/fluxcd/flux/helm-0.10.1/deploy-helm/flux-helm-release-crd.yaml
```

File: docs/tutorials/get-started-helm.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ helm repo add fluxcd https://charts.fluxcd.io
 Apply the Helm Release CRD:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/fluxcd/flux/master/deploy-helm/flux-helm-release-crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/fluxcd/helm-operator/master/deploy/flux-helm-release-crd.yaml
```
Comment (stefanprodan) [inline docs/tutorials/get-started-helm.md]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/fluxcd/flux/helm-0.10.1/deploy-helm/flux-helm-release-crd.yaml
```

**Comment by stefanprodan** [comment]
The chart readme has been updated in #2367 and the get started guide in [9e3bdaf](https://github.com/fluxcd/helm-operator-get-started/commit/9e3bdafb44866b587ac95fec670da7bf6440ee5b)

**Pull request merged**